src/lib.rs

```diff
@@ -98,7 +98,6 @@ pub mod jellyfish {
     /// Tags to be used for [`tagged-base64`](https://github.com/espressosystems/tagged-base64)
     pub mod tag {
         pub const VERKEY: &str = "VERKEY";
-        pub const SIGNKEYPAIR: &str = "SIGNKEYPAIR";
         pub const SIG: &str = "SIG";
         pub const NODE: &str = "NODE";
```

Comment:
Also, as far as I can see, the `NODE` and `LEAF` tags are not used. Should we remove them?

Reply:
I've also removed `SIG`, since it's now replaced by `SCHNORR_SIG`.

Reply:
just curious, how are you able to check their usage throughout all dependents?

Reply:
Since it would be defined in the jellyfish lib (if it were defined), if nothing is tagged there with these tags, then there can't be any dependency using it right?